docs/sources/operations/automatic-stream-sharding.md

```diff
@@ -11,6 +11,28 @@ Automatic stream sharding will attempt to keep streams under a `desired_rate` by
 existing streams. When properly tuned, this should eliminate issues where log producers are rate limited due to the
 per-stream rate limit.
 
+**To enable automatic stream sharding:**
+1. Edit the global [limits_config]({{<relref "../../configuration/#limits_config">}}) of the Loki configuration file:
+   ```yaml
+   limits_config:
+     shard_streams:
+         enabled: true
+   ```
+2. Optionally change the `desired_rate` in bytes if ???:
```

Comment:
@MasslessParticle when should one change the default? Also, what's the unit? Is there a reason we don't support strings such as `5MB` like in other parts of the configuration?

Reply:
Users want to make `desired_rate` lower if they find themselves hitting the `per_stream_rate_limit`. The default is 3MB which should (more or less) keep them under the default PSRL. 

This config does support stings. It is expressed as a  human-readable size. e.g. `5MB`, `1024KB`, `6` etc. If no unit is provided, the unit is bytes